lib/update.js

```diff
@@ -16,16 +16,19 @@ var request = require('request')
 /**
  * Update the regexp.js file
  *
+ * @param {String} customRemote Custom resource URI (default remote is used if not a string or falsy)
  * @param {Function} callback Completion callback.
  * @api public
  */
-exports.update = function update(callback) {
+exports.update = function update(customRemote, callback) {
+  var useCustomRemote = customRemote && typeof customRemote === 'string';
+
   // Prepend local additions that are missing from the source
   fs.readFile(exports.before, 'utf8', function reading(err, before) {
     if (err) return callback(err);
 
     // Fetch the remote resource as that is frequently updated
-    request(exports.remote, function downloading(err, res, remote) {
+    request(useCustomRemote ? customRemote : exports.remote, function downloading(err, res, remote) {
```

Comment:
If I do that it will be a breaking change and people using `useragent(true)` will have an error at request